samples/drivers/lora/send/README.rst

```diff
@@ -0,0 +1,36 @@
+.. zephyr:code-sample:: lora
+   :name: lora send
+   :relevant-api: lora_radio
+
+   Transmit a preconfigured payload every second using the lora radio.
```

Comment:
Several suggestions below:

- name the sample so that it's easier to add lora-receive later on
- Spell "LoRa", not "lora"
- `relevant-api` is meant to be a list of existing Doxygen groups.

```suggestion
.. zephyr:code-sample:: lora-send
   :name: LoRa send
   :relevant-api: lora_api

   Transmit a preconfigured payload every second using the lora radio.
```

Reply:
Apologies, I don't understand how naming the sample with a hyphen makes it easier to find the receive sample. Could you please clarify?

Reply:
> Apologies, I don't understand how naming the sample with a hyphen makes it easier to find the receive sample. Could you please clarify?

my point is that soon enough there will be a proper documentation for "LoRa receive" sample as well, right? And since it will need a unique name, I'm guessing we'll likely go for "lora-receive", so we might as well already make sure the send sample as a similar naming. Makes sense?

Reply:
That's fine - I just didn't understand whether the inclusion of white-space in the name was a problem. I will make the change.

Reply:
Oh now I see where the confusion comes from: I suggested two different things: making the "id" unique and hyphenated, *and* changing the name so that "LoRa" is capitalized :)